netlify.toml

```diff
@@ -0,0 +1,3 @@
+[build]
```

Comment:
have you included the redirects file?


https://www.netlify.com/docs/redirects/

we need to handle pushState routes.

Reply:
https://github.com/peachcloud/peachhub-provider/blob/master/dist/_redirects

Reply:
included in the `netlify.toml` file